src/officeGraph.ts

```diff
@@ -19,11 +19,11 @@ export class OfficeGraph {
                 let data = await response.json();
                 return resolve(data);
             }
-            return reject(new Error(`GET for ${url} failed with ${response.status} ${response.statusText}`));
+            return reject(new Error(`GET for ${url} failed with ${response.status} ${response.statusText} and token ${accessToken.substring(0, 5)}`));
```

Comment:
would a fixed separator between status and statusText help with reading?

Reply:
same lower down also.

Reply:
status is an http status number like 404 and status text is something like "not found".  

GET for http://... failed with 400 not found...